contrib/buildrpm.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+#set -vx
+
+PROGRAM="libs2n"
+VERSION="20150703"
```

Comment:
Until we have a release, is there a way to make this the commit id? or does the number have to increase?